cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -662,4 +668,9 @@ protected Future<Boolean> delete(Reconciliation reconciliation) {
         return withIgnoreRbacError(reconciliation, clusterRoleBindingOperations.reconcile(reconciliation, KafkaResources.initContainerClusterRoleBindingName(reconciliation.name(), reconciliation.namespace()), null), null)
                 .map(Boolean.FALSE); // Return FALSE since other resources are still deleted by garbage collection
     }
+
+    public void setClock(Clock clock) {
+        this.clock = clock;
+    }
+
```

Comment:
I don't think this is really needed anymore. It seems to have only one place where it is used -> in `CertificateRenewalTest`. But it is not needed there either because the `clock` instance is passed directly to `reconcileCas`. So I think we can remove this method and the call to it.

Reply:
Ack. I removed it.